frontend/src/metabase-lib/filter.ts

```diff
@@ -18,71 +14,14 @@ export function filterableColumns(
   return ML.filterable_columns(query, stageIndex);
 }
 
-export function filterableColumnOperators(
-  filterableColumn: ColumnWithOperators,
-): FilterOperator[] {
-  return ML.filterable_column_operators(filterableColumn);
-}
```

Comment:
I think we should keep `filterableColumnOperators`, so it's not FE's responsibility to decide which operators are available for a given column. WDYT?

Reply:
We won't be able to use it in many places. E.g. we special-case most operators for dates, also for booleans etc

Reply:
Also, the thing is that "column operators" is not a thing anymore with raw expressions